packages/salesforce-adapter/test/filters/utils.test.ts

```diff
@@ -1405,4 +1407,28 @@ describe('filter utils', () => {
       })
     })
   })
+
+  describe('isCustomField', () => {
+    it('should return true for custom fields', () => {
+      expect(isCustomField(mockTypes.User.fields.Manager__c)).toBeTrue()
```

Comment:
```suggestion
      expect(mockTypes.User.fields.Manager__c).toSatisfy(isCustomField)
```
nit: Consider the following styling from jest-extended for more indicative matcher error messages